gapic/templates/%namespace/%name_%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -139,26 +139,39 @@ def _get_http_options():
     return response
 {% endmacro %}
 
-{% macro rest_call_method_common(body_spec, method_name, service_name) %}
+{% macro rest_call_method_common(body_spec, method_name, service_name, is_async=False) %}
+{% set await_prefix = "await " if is_async else "" %}
+{% set async_prefix = "async " if is_async else "" %}
```

Comment:
I couldn't determine if this is used. Remove if it's not needed.

Reply:
If `async_prefix = "async "` is unused, I would recommend to remove it. I couldn't find `async_prefix` in the macro other than the location where it is defined

Reply:
Apologies, I missed that! You're right and that's a good catch. I've removed it now!